node/engine/messageservice/p2p-message-service/service.go

```diff
@@ -128,9 +127,9 @@ func (ms *P2PMessageService) setupMdns() {
 func (ms *P2PMessageService) setupDht(bootPeers []string) {
 	ctx := context.Background()
 	var options []dht.Option
-	if len(bootPeers) == 0 {
-		options = append(options, dht.Mode(dht.ModeServer)) // Server mode: allows other peers to connect to this node
-	}
+	options = append(options, dht.BucketSize(20))
```

Comment:
Is there any reason for a bucket size of 20?

Reply:
The default value is 20. I just set this option so it is obvious which bucket size we are using and it is easy to change later if we want.